From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@v-kagri__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-kagri). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

jeevansd committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

jeevansd committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr

jeevansd committed:
Adding the new article on Advance Certificate options in Enterprise Apps

jeevansd committed:
Adding the new article on Advance Certificate options in Enterprise Apps

Commit:
edit pass: azure-stack-kv-sample-app

Commit:
added c# c# device mgmt article

Commit:
added device register import

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into csharp-csharp-device-mgmt-article
sync with master

paulathurman committed:
edit pass: two cognitive-services-custom-speech articles

paulathurman committed:
edit pass: two cognitive-services-custom-speech articles

paulathurman committed:
edit pass: one cognitive-services-custom-speech article

itechedit committed:
edit pass: hdinsight-hadoop-use-sqoop-powershell

jeevansd committed:
Adding the new article on Advance Certificate options in Enterprise Apps

jeevansd committed:
Adding the new article on Advance Certificate options in Enterprise Apps

itechedit committed:
Update hdinsight-use-latest-powershell.md

itechedit committed:
Update hdinsight-hadoop-use-sqoop-powershell.md

itechedit committed:
Update hdinsight-hadoop-use-sqoop-powershell.md

itechedit committed:
Update hdinsight-use-latest-powershell.md

Commit:
edit pass: event-schema

LoriMerrick committed:
Made a small edit.

LoriMerrick committed:
Update event-schema.md

itechedit committed:
Update hdinsight-use-latest-powershell.md

itechedit committed:
Update hdinsight-use-latest-powershell.md

itechedit committed:
Update hdinsight-hadoop-use-sqoop-powershell.md

itechedit committed:
edit pass: app-service-linux-faq

itechedit committed:
Update hdinsight-hadoop-use-sqoop-powershell.md

itechedit committed:
Update app-service-linux-faq.md

itechedit committed:
Update hdinsight-hadoop-use-sqoop-powershell.md

itechedit committed:
Update hdinsight-use-latest-powershell.md

itechedit committed:
Update hdinsight-hadoop-use-sqoop-powershell.md

itechedit committed:
Update hdinsight-use-latest-powershell.md

itechedit committed:
Update hdinsight-use-latest-powershell.md

ktoliver committed:
Update hdinsight-hadoop-use-sqoop-powershell.md

itechedit committed:
Update hdinsight-hadoop-use-sqoop-powershell.md

itechedit committed:
Update hdinsight-hadoop-use-sqoop-powershell.md

Commit:
updates to csharp article

saghorpa committed:
few script changes

itechedit committed:
Update hdinsight-hadoop-use-sqoop-powershell.md

itechedit committed:
removing file

itechedit committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into hdinsight-hadoop-use-sqoop-powershell

itechedit committed:
fixing merge conflicts

itechedit committed:
removing file

Commit:
updated media files, selector, toc, and csharp article

Commit:
code bug and image name fix

Commit:
fixed bad image reference

Commit:
image fname fix and hopefully 1 px border

Commit:
outlined and marked up image files

Commit:
couple of edits at article top

Commit:
added github account to metadata to fix warning

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into event-schema

Commit:
edit pass: event-schema

Commit:
syncing changes

Commit:
edit pass: event-schema

Commit:
edit pass: event-schema

itechedit committed:
Update app-service-linux-faq.md

saghorpa committed:
update for boot lun script description

saghorpa committed:
update for boot lun script description

saghorpa committed:
update for boot lun script description

Commit:
fixed image files and added combined run output image -- also minor text tweaks

Commit:
removed old app and service run image files

TerryLanfear committed:
updates

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into sept-mail

Commit:
added note about polling for reporeted properties

Commit:
got rid of extraneous jobClient variable in TriggerReboot

Commit:
Move limits on intents, entities, phrase lists, and utterances to the FAQ

ecfan committed:
Updates for CFR and MVC

Commit:
Fixes to language to Improve acrolinx score

Commit:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into csharp-csharp-device-mgmt-article
sync with master before merge

TerryLanfear committed:
final fix

TerryLanfear committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into sept-mail

ecfan committed:
Acrolinx issues and other tweaks

ecfan committed:
Acrolinx fixes

ecfan committed:
Formatting fixes

ecfan committed:
Acrolinx fixes

ecfan committed:
New and updated art

barclayn committed:
minor changes per Steve

ecfan committed:
Add art and tweaks

ecfan committed:
Tweaks

ecfan committed:
Add art and tweaks

ecfan committed:
Fix Acrolinx, tweaks

ecfan committed:
Add art

ecfan committed:
Fix typo

ecfan committed:
Fix typo

ecfan committed:
Fix typo

ecfan committed:
Tweaks

ecfan committed:
Acrolinx fix

ecfan committed:
Tweaks

Commit:
Added details for REST API results

1) Incorporating review comments from Lisaco88
2) Actual changes adding more details for REST API

eamonoreilly committed:
Adding billing information

eamonoreilly committed:
Adding new image

eamonoreilly committed:
Moved billing information to top section

eamonoreilly committed:
Added Note about pricing for Azure VMs

ecfan committed:
Tweak

Commit:
remove utterance number limit - no longer 2000

Commit:
Merge branch 'master' into limits-9-15

ShawnJackson committed:
Update azure-stack-kv-sample-app.md

ShawnJackson committed:
Merge pull request #21922 from LoriMerrick/azure-stack-kv-sample-app2

edit pass: azure-stack-kv-sample-app

v-shils committed:
Merge pull request #22910 from TerryLanfear/sept-mail

Sept mail

ShawnJackson committed:
Update hdinsight-hadoop-use-sqoop-powershell.md

Commit:
Merge pull request #23007 from novino/patch-5

Added details for REST API results

ShawnJackson committed:
Merge pull request #22211 from itechedit/hdinsight-hadoop-use-sqoop-powershell

edit pass: hdinsight-hadoop-use-sqoop-powershell

mahesh-unnikrishnan committed:
Update active-directory-ds-getting-started-password-sync.md

ShawnJackson committed:
Update event-schema.md

tfitzmac committed:
added Flow as handler

Juliako committed:
Update media-services-face-and-emotion-detection.md

paulathurman committed:
Update cognitive-services-custom-speech-subscribe.md

ShawnJackson committed:
Merge pull request #22574 from LoriMerrick/event-schema

edit pass: event-schema

saghorpa committed:
Update hana-overview-high-availability-disaster-recovery.md

ShawnJackson committed:
Merge pull request #22595 from itechedit/app-service-linux-faq

edit pass: app-service-linux-faq

PRMerger17 committed:
Merge pull request #23054 from Juliako/patch-1

Update media-services-face-and-emotion-detection.md

PRMerger16 committed:
Merge pull request #23053 from tfitzmac/0918eg1

added Flow as handler

PRMerger10 committed:
Merge pull request #22741 from saghorpa/HADRscriptupdate09142017

few script changes

v-shils committed:
Merge pull request #23016 from eamonoreilly/master

Updated documentation to reflect pricing change for Azure VMs and new onboarding experience from the portal

PRMerger12 committed:
Merge pull request #23051 from mahesh-unnikrishnan/patch-15

Update active-directory-ds-getting-started-password-sync.md

ShawnJackson committed:
Update cognitive-services-custom-speech-subscribe.md

ShawnJackson committed:
Merge pull request #21988 from paulathurman/cognitive-services-custom-speech-subscribe

edit pass: two cognitive-services-custom-speech articles

tfitzmac committed:
added size limits

v-thepet committed:
Merge pull request #22979 from barclayn/pim-updates-steve

minor changes per Steve

v-shils committed:
Merge pull request #22948 from DeniseMak/limits-9-15

Move limits on intents, entities, phrase lists, and utterances to FAQ

PRMerger12 committed:
Merge pull request #23070 from tfitzmac/0918eg3

added size limits

v-thepet committed:
Merge pull request #22952 from ecfan/first-logic-app

Updates for CFR and MVC prep

jeevansd committed:
Adding the new article for Enterprise app advanced certificate options

v-thepet committed:
Merge pull request #21984 from JimacoMS2/csharp-csharp-device-mgmt-article

Csharp csharp device mgmt article

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

GitHubber17 committed:
Merge pull request #21782 from jeevansd/App-Advance-Certificate

App advance certificate

v-alje committed:
Merge pull request #23089 from MicrosoftDocs/FromPublicRepo

From public repo
